src/List/ListItem.spec.js

```diff
@@ -201,4 +201,29 @@ describe('<ListItem />', () => {
       assert.strictEqual(wrapper.state().hovered, false, 'should reset the state');
     });
   });
+
+  describe('containerElement', () => {
```

Comment:
We have a convention where we prefix the name with `prop: ` to describe the property expectations.